modules/cnf-installing-clusters.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+// Epic CNF-9657 (TELCODOCS-1540) (4.16)
+// * edge_computing/cnf-understanding-siteconfig-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cnf-installing-clusters_{context}"]
+= Installing {sno} clusters with the {sco}
+
+Install your clusters with the {sco} by using the default cluster templates.
+
+.Prerequisites
+
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+* If using {ztp}, you have configured your {ztp} environment.
+* You have the default cluster templates.
+//Anything else?
+
+.Procedure
+
+. Create your cluster and node templates for the Image-based Install Operator by running the following commands:
++
+--
+[source,terminal]
+----
+$ oc apply -f ibi-cluster-templates-v1.yaml
+----
+
+[source,terminal]
+----
+$ oc apply -f ibi-node-templates-v1.yaml
+----
+--
+
+. Create the `ClusterInstance` CR, named for example `clusterinstance-ibi.yaml`, and reference your templates:
```

Comment:
Add a step prior to this for creating the target namespace, pull-secret, bmc secret and extra-manifests configmap on the hub cluster.